test/onnx/expect/TestOperators.test_remainder.expect

```diff
@@ -3,41 +3,30 @@ producer_name: "pytorch"
 producer_version: "CURRENT_VERSION"
 graph {
   node {
+    input: "0"
     input: "1"
     output: "2"
-    name: "Cast_0"
-    op_type: "Cast"
-    attribute {
-      name: "to"
-      i: 1
-      type: INT
-    }
+    name: "Div_0"
+    op_type: "Div"
```

Comment:
Hi @BowenBao 
I updated the expected IR of remainder, since the operator of `remainder` is changed. And the `test/onnx/test_operator.py  TestOperators.test_remainder` is passed now.